### ISSUE TITLE
Use "starting _" when rescheduling recurring tasks

### ISSUE THREAD

**Issue by alanvardy** [alanvardy/tod#428]
When a task is recurring and gets rescheduled it wipes out the recurring data and makes it a one time task

either

1. Find a way to reschedule without impacting the next recurring appointment, or
2. Skip scheduling recurring tasks

**Comment by stacksjb** [comment]
The official app differentiates between "Reschedule" and "postpone".

 I will have to look into the API to see if it allows that differentiation.

**Comment by alanvardy** [comment]
Interesting, I wasn't able to find anything in the docs, so for now I am just disabling the rescheduling of recurring tasks.

It caused me to lose some recurring tasks and I wouldn't want it to happen to anyone else!

**Comment by stacksjb** [comment]
I agree - that would be an issue for me too! 

I talked to Omar at Todoist and we figured out that while the official app does "postpone", that differentiation doesn't exist within the API (they are considering adding it, but doesn't exist right now). 

They recommended setting recurring tasks using a "starting <x>" format.
For example, if I had a task recurring every Sunday, and I decided to reschedule it to a Tuesday, setting to be due "every Sunday starting Tuesday" would move it to the next Tuesday for the next occurrence, but recurring every Sunday thereafter.
